tests/child.cpp

```diff
@@ -20,11 +20,30 @@ namespace bpftrace::test::child {
 
 using ::testing::HasSubstr;
 
-#define TEST_BIN "/bin/ls"
-#define TEST_BIN_ERR "/bin/ls /does/not/exist/abc"
-#define TEST_BIN_SLOW "/bin/sleep 10"
+class childproc : public ::testing::Test {
+protected:
+  void find(std::string &out, const char *path)
+  {
+    std::error_code ec;
+    auto self = std::filesystem::read_symlink("/proc/self/exe", ec);
```

Comment:
See `src/filesystem.h`. Not sure if this is still necessary since we c++20 now, but probably good to keep it consistent and clean it all up in one PR.
```suggestion
    auto self = std_filesystem::read_symlink("/proc/self/exe", ec);
```

Reply:
I'd rather just resolve this bit of debt. That header doesn't make sense anymore. Will send  #3722 for review shortly.